webpack.config.babel.js

```diff
@@ -33,14 +34,13 @@ module.exports = (env, argv) => ({
   module: {
     rules: [
       {
-        include: path.join(__dirname, 'src/lib'),
         test: /\.(js|jsx)$/,
         use: [{ loader: 'babel-loader' }],
       },
       {
         test: /\.scss$/,
         use: [
-          { loader: 'style-loader' },
```

Comment:
I just realized, that by removing the `style-loader` loader, there is no way to use prebuild RUI with inline styles (inserted into <head> instead of linking a CSS file). Inline styles are what we are using in RACOM.

We need to support both ways.
* with CSS files for the docs
* with inline CSS for projects

Reply:
Agreed on video call, that the present state is OK.